src/rust/ide/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -44,16 +48,15 @@ const CORNER_RADIUS : f32       = super::super::node::CORNER_RADIUS;
 pub mod background {
     use super::*;
 
-    // TODO use style
     ensogl::define_shape_system! {
         (selected:f32,radius:f32,roundness:f32) {
             let width  : Var<Pixels> = "input_size.x".into();
             let height : Var<Pixels> = "input_size.y".into();
-            let radius        = 1.px() * &radius;
-            let color_bg      = color::Lcha::new(0.2,0.013,0.18,1.0);
-            let corner_radius = &radius * &roundness;
-            let background    = Rect((&width,&height)).corners_radius(&corner_radius);
-            let background    = background.fill(color::Rgba::from(color_bg));
+            let radius               = 1.px() * &radius;
+            let color_transp         = color::Rgba::new(1.0,0.0,0.0,0.000_000_1);
```

Comment:
Here should be TODO explaining why we make it transparent. And create an issue to track this and put here the link/number.